CourseGrab/ViewControllers/Search/SearchTableViewCell.swift

```diff
@@ -0,0 +1,137 @@
+
+        trackingButton.isHidden = true
+        trackingButton.layer.cornerRadius = 2
+        trackingButton.layer.borderWidth = 1
+        trackingButton.layer.borderColor = UIColor.courseGrabRuby.cgColor
+        trackingButton.titleLabel?.font = ._12Semibold
+        trackingButton.setTitle("REMOVE", for: .normal)
+        trackingButton.setTitleColor(.courseGrabRuby, for: .normal)
+        trackingButton.layer.borderColor = UIColor.courseGrabRuby.cgColor
+        containerView.addSubview(trackingButton)
+
+        statusBadge.contentMode = .scaleAspectFit
+        containerView.addSubview(statusBadge)
+        statusBadge.isHidden = true
+
+        subtitleLabel.font = ._14Semibold
+        containerView.addSubview(subtitleLabel)
+        subtitleLabel.isHidden = true
+
+        // Setup constraints
+
+        arrowImageView.snp.makeConstraints { make in
+            make.size.equalTo(16)
+            make.centerY.equalTo(titleLabel)
+            make.trailing.equalToSuperview().inset(16)
+        }
+
+        containerView.snp.makeConstraints { make in
+            make.top.bottom.equalToSuperview().inset(6)
+            make.leading.equalTo(20)
+            make.trailing.equalToSuperview().inset(20)
+        }
+
+        titleLabel.snp.makeConstraints { make in
+            make.trailing.equalTo(arrowImageView.snp.leading).offset(-8)
+            make.top.bottom.leading.equalToSuperview().inset(16)
+        }
+
+    }
+
+    func configure(for section: Section) {
+        titleLabel.text = section.title
+
+        if section.isTracking {
+            divider.isHidden = false
+            trackingButton.isHidden = false
+            statusBadge.isHidden = false
+            statusBadge.image = section.status.icon
+            subtitleLabel.isHidden = false
+            subtitleLabel.text = section.section
+
+            titleLabel.snp.remakeConstraints { make in
+                make.leading.top.equalToSuperview().inset(16)
+                make.trailing.equalTo(arrowImageView.snp.leading).offset(-8)
+                make.bottom.equalTo(divider.snp.top).offset(-11)
+            }
+
+            divider.snp.makeConstraints { make in
+                make.width.equalToSuperview()
+                make.height.equalTo(1)
+                make.top.equalTo(titleLabel.snp.bottom).offset(11)
+                make.bottom.equalToSuperview().inset(44)
+            }
+
+            statusBadge.snp.makeConstraints { make in
+                make.size.equalTo(16)
+                make.leading.equalToSuperview().offset(16)
+                make.top.equalTo(divider.snp.bottom).offset(12)
+                make.bottom.equalToSuperview().inset(16)
+            }
+
+            subtitleLabel.snp.makeConstraints { make in
+                make.leading.equalTo(statusBadge.snp.trailing).offset(12)
+                make.centerY.equalTo(statusBadge.snp.centerY)
+            }
+
+            trackingButton.snp.makeConstraints { make in
```

Comment:
why are these constraints being placed here? the only times you should creating constraints in the `configure` are when you have to update them based on the `data` that is being used to configure this cell. if the constraints stay the same, we should just `makeConstraints` inside the `init`

Reply:
When I move them to `init` we are getting a ton of constraint conflicts

Reply:
what do the conflicts say?

Reply:
<img width="1008" alt="Screen Shot 2020-02-11 at 1 56 21 PM" src="https://user-images.githubusercontent.com/19507508/74268711-51f5d880-4cd6-11ea-9ab0-f56a4ea54fbe.png">

Reply:
you should try to resolve them, heres a useful site: https://www.wtfautolayout.com
you can past the stuff in between the `(` and `)` and it'll tell you what those conflicts mean.
from a first glance, it seems its because for your views, you define `top`, `bottom`, and `height` when you should only have 2 out of those 3